zilliqa/src/contracts/deposit_v1.sol

```diff
@@ -0,0 +1,291 @@
+// SPDX-License-Identifier: MIT OR Apache-2.0
```

Comment:
Reviewer please compare this file again current main's `zzq2/zilliqa/src/contracts/deposit.sol`